src/world.rs

```diff
@@ -124,6 +127,20 @@ impl World {
         ));
     }
 
+    fn scale_duration(&self, duration: u128) -> f32 {
+        (duration as f32 / 1000f32).clamp(0.0, 1.0)
+    }
+
+    fn draw_power_bar(&self, power: f32) {
+        draw_rectangle(
+            10.0,
+            screen_height() - 10.0,
+            20.0,
+            power * -150.0,
```

Comment:
```suggestion
            power * -10.0,
```

we already multiply it by `15.0` before passing as an argument.

and as you might already know `15 * -10 = -150`